packages/benchmarks/lit-element/list/index.ts

```diff
@@ -0,0 +1,245 @@
+/**
+ * @license
+ * Copyright (c) 2020 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+import {
+  html,
+  LitElement,
+  css,
+  PropertyDeclaration
+} from 'lit-element';
+import {property, customElement} from 'lit-element/lib/decorators.js';
+
+// IE doesn't support URLSearchParams
+const params = document.location.search
+  .slice(1)
+  .split('&')
+  .map((p) => p.split('='))
+  .reduce((p: {[key: string]: any}, [k, v]) => ((p[k] = v || true), p), {});
+
+type SimpleItem = {[index: string]: string};
+
+function makeItem(prefix: number) {
+  let o: SimpleItem = {};
+  for (let i = 0; i < 99; i++) {
+    o['value' + i] = prefix + ': ' + i;
+  }
+  return o;
+}
+
+function generateData(count: number) {
+  let data = [];
+  for (let i = 0; i < count; i++) {
+    data.push(makeItem(i));
+  }
+  return data;
+}
+
+const data = generateData(250);
+const otherData = generateData(500).slice(250);
```

Comment:
Might be nice to pull all the constants used in the benchmark up to the top, just so it's easier to grok/tinker with. There's `updateCount` below also, maybe others?

Reply:
Done.